internal/k8shandler/podtemplate.go

```diff
@@ -94,16 +94,21 @@ func ArePodSpecDifferent(lhs, rhs v1.PodSpec, strictTolerations bool) bool {
 	return changed
 }
 
-// Copies the original template and overwrites the volume in the spec
-// if the overwriting template contains an actively used storage
-func CopyPodTemplateSpec(original, overwrite v1.PodTemplateSpec, overwriteVolume bool) v1.PodTemplateSpec {
-	templateCopy := original
+// IsPodTemplateSpecUpdateable compares two pod templates using ArePodTemplateSpecDifferent.
+// The update pod template is changed so that it reflects certains aspects of the current.
+// This includes things like the volumes.
+func IsPodTemplateSpecUpdatable(current, update v1.PodTemplateSpec) bool {
+	modUpdate := CreateUpdatablePodTemplateSpec(current, update)
```

Comment:
why abstract this to a new function? couldn't we just as easily ignore any spec volume changes in `ArePodTemplateSpecDifferent)` ?

Reply:
I was not sure if that function was being used elsewhere for something, but yeah I could change that to avoid "duplication".